### PR TITLE
Feature/om haffa

### PR DESCRIPTION
**Pull request by rubikuo**
### **Explain the changes you've made**

Removed Message page and added Om page
Created Om page 
Fixed active link for the menu bar( before when we landed on the app for the first time the menu options doesnt show the active on the home option") 
 
### **Explain why these changes are made**

Om page right now only made in front end.  Discussed with Tommy and Maria, right now the "edit" function is not prior as if we want to make it "editable" in different sections, the it's kind of much for for this short time. Need further discussion. 

### **Explain your solution**

Added a new route, changed Link tag to NavLink tag and added activeStyle

### **How to test the changes?**
Check if the Home option in the menu is active once linking to the app on the first time. 
Check if there is Om page according to the design

### CODE REVIEW

File: src/pages/About.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+            grund av corononapandemin tar vi endast emot bokade besök och max
+            två personer per bokning.
+          </p>
+          <p>
+            Du kan mejla aterbruket.amf@helsingborg.se eller ringa om du hittar
+            en möbel i annonserna som du vill reservera. Möbler kan reserveras i
+            två veckor.
+          </p>
+          <p>
+            Ring oss på 042-10 28 00 för att boka en tid om du vill hämta och
```
Comment (JohannaENilsson) [inline src/pages/About.tsx:55]: Change phone number and email to a link

**Review by JohannaENilsson — CHANGES_REQUESTED** [review]
Looks great!

File: src/pages/About.tsx (hunk 1 above)
Comment (rubikuo) [reply src/pages/About.tsx]: Har fixat :)

**Review by annaeklov — CHANGES_REQUESTED** [review]
image
maybe the icon should be a question mark as the design.

image
Should we add the emojis like the design?

**Pull request merged**